### PR TITLE
docs-fr | docs | reference | glossary | certificate

### PR DESCRIPTION
**Pull request by ryanzidago**

### CODE REVIEW

File: content/fr/docs/reference/glossary/certificate.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Certificate
```
Comment (sftim) [inline content/fr/docs/reference/glossary/certificate.md]:
```suggestion
title: Certificat
```

File: content/fr/docs/reference/glossary/certificate.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+date: 2018-04-12
+full_link: /docs/tasks/tls/managing-tls-in-a-cluster/
+short_description: >
+  Fichier cryptograhpiquement sécurisé utilisé pour valider l'accès au cluster Kubernetes.
+
+aka:
+tags:
+- security
+---
+ Fichier cryptograhpiquement sécurisé utilisé pour valider l'accès au cluster Kubernetes.
```
Comment (sftim) [inline content/fr/docs/reference/glossary/certificate.md]:
```suggestion
 Fichier cryptographiquement sécurisé utilisé pour valider l'accès au cluster Kubernetes.
```

**Comment by remyleone** [comment]
@sftim Can you make a new PR?

**Comment by sftim** [comment]
Yep, will do.

**Pull request merged**